### PR TITLE
changelog for 1.2.8

### PR DESCRIPTION
**Pull request by paulhainsworth-chia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,58 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Bumped sortedcontainers to version 2.4.0.
+- Dropped some redundant code in plotting/manager.py
+- Updated some hooks: Update `flake8` to 3.9.2, `pre-commit-hooks` to 4.0.1, `black` to 21.8b0
+- Bump clvm_rs to 0.1.14.
+- Added tests for invalid list terminators in conditions.
+- Updated blspy to 1.0.6.
+- Made a change to allow the host to be configurable for the timelord launcher.
+- Thanks @dkackman for adding the ability to collect the IDs of plots as they are queued and return them with the response.
+- Made the SpendBundle.debug use the default genesis challenge by default.
+- Changes in full node to execute sqlite pragmas only once, at the level where the database is opened, and changed pragma synchronous=FULL to NORMAL to improve disk I/O performance. Also removed redundant database pragmas in wallet.
```
Comment (emlowe) [inline CHANGELOG.md:41]: Actually, we are using  "synchronous=OFF" - I think the code was changed but not the PR comments.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,58 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Made a change to remove CoinStore's dependency on FullBlock to just pass in the parts of the block necessary to add the block.
+- Switched from using asynchio.to_thread to threading.Thread, as to_thread is only available in Python 3.9.
```
Comment (emlowe) [inline CHANGELOG.md:43]: I think you can eliminate this - this was in test code added in main so it doesn't need to be called out here

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,58 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+- Thanks @mgraczyk for the fix to keyring_path.
+- Fixed a issue blocking the Ubuntu installer that required manual installation of Python 3.9 as a workaround.
+- Fixed an issue where the config.yaml and keyring.yaml are only partially written out to, if another process is attempting to read the config while it's being written.
+- Fixed rmtree call in create_pool_plot.
+- Thanks @Knight1 for fixing an issue in which fix-ssl-permissions shows the current 'mode' not the 'updated mode'.
+- Fixed Mypy issues on Python 3.9.7 that generated many errors when running mypy.
+- Thanks @olivernyc for fixing an edge case with negative inputs to 'truncate_to_significant_bits'.
+- Added a fix for Windows installs that were seeing exceptions when writing to the keyring.
+- Fixed a GUI loading issue on the farm page that caused an endless spinner for some users.
```
Comment (emlowe) [inline CHANGELOG.md:59]: I think this was introduced in main post-1.2.7 - and it's not a fix to previous versions?

File: CHANGELOG.md (hunk 1 above)
Comment (wjblanke) [reply CHANGELOG.md:41]: the confusion is because we have dropped two levels in two stages. yes now we are using OFF

File: CHANGELOG.md (hunk 3 above)
Comment (wjblanke) [reply CHANGELOG.md:59]: right it went in both branches so doesn't need to be reannounced

File: CHANGELOG.md (hunk 2 above)
Comment (paulhainsworth-chia) [reply CHANGELOG.md:43]: All four lines here 40-43 @emlowe ?

**Review by emlowe — DISMISSED** [review]
good for me

**Review by hoffmang9 — DISMISSED** [review]
lgtm

**Review by hoffmang9 — APPROVED** [review]
even better

**Review by jack60612 — APPROVED** [review]
Looks good

**Comment by jack60612** [comment]
@paulhainsworth-chia could we squeeze in #8651

**Pull request merged**